hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -99,7 +100,8 @@ public class SingleTransactionRecordBuilderImpl
                 ContractCreateRecordBuilder,
                 ContractCallRecordBuilder,
                 CryptoDeleteRecordBuilder,
-                TokenUpdateRecordBuilder {
+                TokenUpdateRecordBuilder,
+                NodeStakeUpdateRecordBuilder {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Perhaps add an `@Override` annotation on the method coming from this interface.